tools/vagrant/Vagrantfile

```diff
@@ -87,7 +87,7 @@ Vagrant.configure('2') do |config|
       echo PATH=${PATH}:${HOME}/bin:${OPENWHISK_HOME}/tools/build >> /etc/environment
       cd ${OPENWHISK_HOME}
       cd tools/ubuntu-setup
-      su vagrant -c 'source all.sh'
+      su vagrant -c 'source all.sh oracle'
```

Comment:
Why hardcode `oracle` here when the default every where else is `open`?

If you want to allow user to easily switch to use `oracle` make it an envirable variable they can do `JAVA_SOURCE=oracle ./hello`